cinderblock/integrate.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
```

Comment:
It's a bit confusing that `commitstatus.py` maps to the `commitstatus` command (in setup.py), but `integrate.py` maps to `cinderblock`. What do you think about renaming this to `cinderblock/cinderblock.py` for consistency?